Svc/BufferManager/BufferManager.cpp

```diff
@@ -270,10 +270,31 @@ namespace Svc {
 
   }
 
-  // ----------------------------------------------------------------------
+  U32 BufferManager :: getBufferContext(const U32 managerId, const U32 bufferId) {
+      // Context is a concatenation of IDs, so check for overflows
+      FW_ASSERT(managerId <= 0x7FFF, managerId);
```

Comment:
Why < 0x7FFF? You have 16-bits, don't you?

Reply:
Yes, but the ID 0xFFFFFFFF is reserved and that one bit is taken out of the manager ID's space.

Reply:
Ah - that's right.